internal/plugins/ansible/v1/init.go

```diff
@@ -0,0 +1,112 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+Modifications copyright 2020 The Operator-SDK Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ansible
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/pflag"
+	"k8s.io/apimachinery/pkg/util/validation"
+	"sigs.k8s.io/kubebuilder/pkg/model/config"
+	"sigs.k8s.io/kubebuilder/pkg/plugin"
+	"sigs.k8s.io/kubebuilder/pkg/plugin/scaffold"
+
+	"github.com/operator-framework/operator-sdk/internal/kubebuilder/cmdutil"
+	"github.com/operator-framework/operator-sdk/internal/plugins/ansible/v1/scaffolds"
+)
+
+type initPlugin struct {
+	config    *config.Config
+	apiPlugin createAPIPlugin
+
+	doAPIScaffold bool
+
+	// For help text.
+	commandName string
+}
+
+var (
+	_ plugin.Init        = &initPlugin{}
+	_ cmdutil.RunOptions = &initPlugin{}
+)
+
+// TODO(asmacdo) documentation
+func (p *initPlugin) UpdateContext(ctx *plugin.Context) {
+	ctx.Description = `Initialize a new Ansible-based operator project.`
+	p.commandName = ctx.CommandName
+}
+
+func (p *initPlugin) BindFlags(fs *pflag.FlagSet) {
+	fs.SortFlags = false
+	fs.StringVar(&p.config.Domain, "domain", "my.domain", "domain for groups")
+	p.apiPlugin.BindFlags(fs)
```

Comment:
```suggestion
	fs.StringVar(&p.config.ProjectName, "project-name", "", "name of this project, the default being directory name")
	p.apiPlugin.BindFlags(fs)
```

The flag for the projectName

Reply:
I would prefer not to make these changes as suggestions to this PR. Did you create these patches from a branch? If so, maybe a pull request to this branch would let me make sure this all goes in atomically and compiles and everything.

Reply:
No. It is in the master for the helm. And we really need these changes.

Reply:
Here's the PR for Helm: https://github.com/operator-framework/operator-sdk/pull/3438

IMO this can be a follow-up. Let's get the core functionality merged and do small updates rather than do everything in a huge monolithic PR.